.github/renovate.json5

```diff
@@ -5,6 +5,9 @@
     "docker:pinDigests",
     "helpers:pinGitHubActionDigestsToSemver"
   ],
+  "postUpdateOptions" : [
```

Comment:
stole it from https://github.com/open-telemetry/opentelemetry-go/blob/796c2477ca5b213b70260098ca5e17fa614d8ae6/renovate.json#L9, docs - https://docs.renovatebot.com/golang/